second-edition/src/appendix-05-translation.md

```diff
@@ -1,14 +1,13 @@
-## Apéndice E: Traducciones del Libro
+## Appendix E: Translations of the Book
```

Comment:
Coming back to english here! Please check 💃

Reply:
hahahahaha whaaat, let me fix, my way of using branches must be wrong, that's probably what's happening